examples/crypto/crypto.metatags

```diff
@@ -0,0 +1,2 @@
+description: BBE on how to perform different cryptographic operations such as hashing, HMAC generation, checksum generation and digitally signing data.
+keywords: ballerina, ballerina by example, bbe, crypto, hash, hmac, sign, encrypt, decrypt
```

Comment:
Shall we reduce the keywords? The best practice is to have 5-8 keywords

Reply:
Updated with 8 keywords. https://github.com/ballerina-platform/ballerina-distribution/pull/551/commits/f962ab67f771beb9643e82826fbf4e5282cb05ec